### ISSUE TITLE
[BUG REPORT] - Error Code 191 on Adobe Creative Cloud after installation

### ISSUE THREAD

**Issue by anmolkuchhal003** [Atlas-OS/Atlas#867]
### Description

After installing **Adobe Creative Cloud** Error Code 191 is seen, Here is the log file of the error that is seen if that helps. 

[Install.log](https://github.com/Atlas-OS/Atlas/files/12188654/Install.log)

### Steps to reproduce (add screenshots if applicable)

1. Open Adobe Creative Cloud
2. Click on the Cloud Icon

### Expected behavior

The User should be able to sync creative cloud correctly and update

### Actual behavior (add screenshots if applicable)

User sees an error code 191

Creative Cloud Update Failed
Sorry We could not update the app.
[Error Code 191](http://www.adobe.com/go/apps_install_hd_error?trackingid=CZY71YK2&prod)

![image](https://github.com/Atlas-OS/Atlas/assets/48596433/d19a0f77-f8e2-4c39-bf16-8ff699d796f1)

Bug Reproduction:
https://github.com/Atlas-OS/Atlas/assets/48596433/945a6b54-2ba7-46d2-a100-19792dcf26ee

### Atlas Version

Atlas 10 22H2

### Desktop information

- 13th Gen Intel(R) Core(TM) i7-13620H   2.40 GHz
- 16 Gigs of DDR5 Ram
- 1 TB SSD storage Samsung
- GeForce RTX 4050 Laptop GPU - Nvidia Studio Driver - 536.67

### Requisites

- [X] This is not a support issue or a question. For any support, questions or help, join our [Discord server](https://discord.atlasos.net).
- [X] I performed a [cursory search of the issue tracker](https://github.com/Atlas-OS/Atlas/issues?q=is%3Aissue) to avoid opening a duplicate issue.
- [X] I checked the [documentation](https://docs.atlasos.net) to understand that the issue I am reporting is not normal behavior.
- [X] I understand that not filling out this template correctly will lead to the issue being closed.

### Additional content

Note: I have read the documentation (https://docs.atlasos.net/troubleshooting/common-issues/adobe/creative-cloud-error-191/#:~:text=Since%20Adobe%20Creative%20Cloud%20is,turned%20off%20the%20Microsoft%20Store) regarding this could be a UWP issue, I have enabled both UWP as well Microsoft Store and it still did not work.

**Comment by JayXTQ** [comment]
https://docs.atlasos.net/troubleshooting/common-issues/adobe/creative-cloud-error-191/

**Comment by JayXTQ** [comment]
Last time I checked, it worked. I know @whallin has creative cloud so maybe he can see if it still works if possible! If it does then I guess you are doing something wrong on your end. Right now I am away so can’t do much to troubleshoot

**Comment by whallin** [comment]
> No worries and Thanks @JayXTQ ....It would be great if @whallin you could share what you did to correct this. I have enabled the UWP via the script as well Microsoft store is already enabled.

@anmolkuchhal003 Unfortunately, I'm not currently running Atlas, so I can't do any troubleshooting without setting up a VM.

However, a quick look at your attached `Install.log` suggests that UWP is the root cause of your problem. I suppose this confirms **your** error 191 cause.

Looking up potential solutions for error 191, it appears that some people have had success with re-enabling the Windows Firewall if it had previously been disabled. Maybe it's worth a shot if you've disabled it?

**Comment by whallin** [comment]
> @whallin I enabled firewall and added some custom rules regarding creative cloud as well, unfortunately that did not help :( I will be switching from atlas till then as I need to use Mocha which I have no idea why but authenticates Adobe for this 191 error and does not let me use it.

@anmolkuchhal003 would you be able to obtain the Adobe log for us again in that case?

**Comment by whallin** [comment]
> Yes sure, any specific requirements? @whallin

@anmolkuchhal003 no, just the install logs :)

**Comment by whallin** [comment]
@anmolkuchhal003 to keep you and everyone else updated, I'm not seeing this error on a fresh install of AtlasOS on Windows 11 using the builds from GitHub actions.

According to the various attached error logs, I believe the error was still caused by UWP/Store being disabled. Unless @he3als or @Xyueta have another theory.

TLDR; The error does not appear on a fresh Windows 11 AtlasOS installation.

**Comment by Xyueta** [comment]
the uwp or store scripts were removed, thus the services couldn't be disabled/removed unless the user changed them manually.

**Comment by whallin** [comment]
On 2023-08-17 19:31, Anmol Kuchhal wrote:
> As I was mentioning I am reinstalling it on a secondary laptop as we
> are speaking, will that work?

Had to delete my previous response as it contained a bunch of junk.

Please try to install Adobe Creative Cloud as soon as you finish 
reinstalling & installing the necessary drivers. Disable nothing except 
what AtlasOS already disables by default.

**Comment by whallin** [comment]
On 2023-08-17 19:37, Anmol Kuchhal wrote:
> @whallin I should use the UWP script right?

You should **not** disable anything related to the Microsoft Store or 
UWP.
